cp-subsystem/fenced-lock-using-fencing-tokens/src/main/java/com/hazelcast/codesamples/cp/fencedlock/FencingOffStaleLockHolders.java

```diff
@@ -132,7 +132,6 @@ public static void main(String[] args) throws InterruptedException, ExecutionExc
 
         // always destroy CP Subsystem data structures otherwise it can lead to a memory leak
         hz1Lock.destroy();
-        hz2Lock.destroy();
```

Comment:
FencedLock hz1Lock = hz1.getCPSubsystem().getLock(lockName);
        FencedLock hz2Lock = hz2.getCPSubsystem().getLock(lockName);

Don't we have two to destroy here? Are you sure we need to only destroy only 1?

Reply:
Although we have two lock objects, the underlying reference is to the same lock in the CP Subsystem. 
Do we need to destroy both or only one?